addons/l10n_at/data/account_tax_report_data.xml

```diff
@@ -12,15 +12,21 @@
                 <field name="name">Balance</field>
```

Comment:
```suggestion
                <field name="name">Base</field>
```
Shouldn't we rename it to "Base" to make the distinction clearer with the "VAT" column?
Also, we should probably add the translation for the column names as well :de: